doc/ssl/SSL_get_version.pod

```diff
@@ -17,6 +17,10 @@ SSL_get_version, SSL_is_dtls - get the protocol information of a connection
 SSL_get_version() returns the name of the protocol used for the
 connection B<ssl>.
 
+The above function should only be called after the intial handshake has been
+completed. Prior to that the results returned from this function may be
+unreliable.
```

Comment:
Same comment here as for the [1.0.2 PR](https://github.com/openssl/openssl/pull/6015#pullrequestreview-113609883)

Reply:
Fixed